src/client/opamArg.ml

```diff
@@ -1505,12 +1510,17 @@ let build_options cli =
        you installed the required dependency by hand. Implies \
        $(b,--assume-depexts), and stores the exceptions upon success as well."
   in
+  let verbose_on =
+    mk_opt ~cli (cli_from cli2_2) ~section ["verbose-on"] "PACKAGES"
+      "Be more verbose on specific packages."
+      Arg.(some (list package)) None ~vopt:(Some [])
```

Comment:
I think it would be better to allow full package name + version as it is currently. This way the feature would be much simpler to use, especially in CI systems where this would be most useful

Reply:
If we have name.version, it can be used only if user knows in advance which version of the package will be installed, which is not always the case.
In usual CI, you may want to have install log of the package you are testing (usually yours), and its inner version can change.
What do you have in mind about disadvantages of only name?

Reply:
ok fair, let's do name only for now, we can always add name.version later if we want or if requested

Reply:
It could evolve into name.optional-version, like that we have all advantages :)